### PR TITLE
addded: navigation component

### PR DESCRIPTION
**Pull request by rkapoor10**
Added navigation component

- for desktop
- for mobile

nav - component

### CODE REVIEW

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.fix-width-50{
+    width: 50rem;
+}
+
+.fix-width-25{
+    width: 50rem;
```
Comment (paarthbajaj) [inline components/navigation/navigation.css]: fix-width-25,  and fix-width-50 have the same CSS property. You can do it in one I think.

**Review by paarthbajaj — APPROVED** [review]
Look great, just see my comment and use prettier if you haven't.

File: components/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
+
+
```
Comment (JantuDeb) [inline components/navigation/navigation.css:3]: remove extra spaces

File: stylesheets/utility.css [hunk 3]
```diff
@@ -193,4 +193,10 @@ h5{
     width: 100%;
 }
 
+.fix-width-50{
+    width: 50rem;
+}
 
+.fix-width-30{
+    width: 30rem;
+}
```
Comment (JantuDeb) [inline stylesheets/utility.css:202]: You can remove `fix` `width-30` is self-explanatory.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes

File: components/navigation/navigation.css (hunk 2 above)
Comment (rkapoor10) [reply components/navigation/navigation.css:3]: done

File: stylesheets/utility.css (hunk 3 above)
Comment (rkapoor10) [reply stylesheets/utility.css:202]: okay

**Pull request merged**